src/orchard/namespace.clj

```diff
@@ -135,6 +167,46 @@
        (filter identity)
        (classpath-namespaces)))
 
+(defn project-jvm-ns-forms
+  "Returns all JVM Clojure `ns` forms defined in sources within the current project,
+  indexed by `ns-name`."
+  []
+  (let [resources (->> (cp/classpath)
+                       (pmap (fn [x]
+                               (when ((every-pred misc/directory? in-project?) x)
+                                 x)))
+                       (filter identity))]
+    (into {}
+          (map (fn [ns-form]
+                 [(ns-form->ns-name ns-form)
+                  ns-form]))
+          (classpath-namespaces resources jvm-clojure-resource-name->ns-form))))
+
+(defn ns-form->imports
```

Comment:
That's not really a conversion function, so I'd probably name it something like `ns-imports` or `get/extract-ns-imports`.

Reply:
You might want to expand on what is / is not a conversion function here https://github.com/bbatsov/clojure-style-guide#naming-conversion-functions

Reply:
Renamed